source/instructors/authoring/assessments/edit-assessment-points.rst

```diff
@@ -7,9 +7,11 @@ Edit Assessment Points
 ======================
 To edit assessment points, follow these steps:
 
-1. In the Guide Editor, click the **Assessments** button to view the list of all assessments.
+1. In the Guide Editor, click the **Assessments** button.
 
-2. Click the assessment to open it and modify the points.
+2. Then click **"View Existing Assessments"** in the bottom right corner to view the list of all assessments.
+
+3. Click the assessment to open it and modify the points.
```

Comment:
these instructions are actually wrong - you can edit the points on the screen that opens and change them. Then you click close in the lower left to close that screen. The screen shot correctly highlights where you can change the points but doesn't show the close button.